include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3831,8 +3826,7 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
 
   // Case 3: standard operation with one index per degree of freedom -> go on
   // here
-  constexpr unsigned int n_vectorization =
-    VectorizedArrayType::n_array_elements;
+  constexpr unsigned int n_vectorization = VectorizedArrayType::size();
```

Comment:
Looking through these changes I realize that we use lots of different abbreviations - at some point (not necessarily this PR) we should use a more uniform style for these short names; my favorite right now is `n_simd` or `n_lanes`.